Content.Server/Reflect/ReflectComponent.cs

```diff
@@ -0,0 +1,25 @@
+using Robust.Shared.Audio;
+
+namespace Content.Server.Reflect;
```

Comment:
should the `Reflect` namespace be moved to `Weapons.Reflect` instead? @metalgearsloth

Reply:
Yeah this should all be under weapons.